config/initializers/gem-plugin_config/sanitizer_config.rb

```diff
@@ -58,5 +58,16 @@ module Config
 
       env[:node]["open"] = "open" if env[:node].has_attribute?("open")
     end
+
+    # On img elements, convert relative paths to absolute:
+    RELATIVE_IMAGE_PATH_TRANSFORMER = lambda do |env|
+      return unless env[:node_name] == "img" && env[:node]["src"]
+
+      base = "https://#{ArchiveConfig.APP_HOST}/"
```

Comment:
Out of curiosity, why does this use `APP_HOST` instead of `APP_URL`?

Reply:
TBH, I didn't remember it was an option. I have `APP_HOST` overridden in my `local.yml`, so it's on my radar, but `APP_URL` is currently only used in scripts that I don't use, so I've never had to override it.